scripts/album.js

```diff
@@ -51,7 +51,58 @@ var createSongRow = function(songNumber, songName, songLength) {
   + '</tr>'
   ;
 
-  return $(template);
+  var $row = $(template);
+
+  var clickHandler = function() {
+    var $songItem = $(this).find('.song-item-number');
```

Comment:
This is different than the checkpoint. I think it's better to place the listener on the higher-level `$row` and find the `.song-item-number`) from there. But, you know, it doesn't work :grin:.